### PR TITLE
Remove redundant sequence expression

### PR DESCRIPTION
**Pull request by cartermp**
This addresses #1101.

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/lists?branch=pr-en-us-1105](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/lists?branch=pr-en-us-1105)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C2%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610022218374672-1105%5Cworkflow_report.html).

File: docs/fsharp/language-reference/lists.md [hunk 1]
```diff
@@ -36,19 +36,13 @@ You can also define list elements by using a range indicated by integers separat
...
-You can also define a list by using a looping construct, as in the following code.
-
-[!code-fsharp[Main](../../../samples/snippets/fsharp/lang-ref-1/snippet1303.fs)]
-
 An empty list is specified by a pair of square brackets with nothing in between them.
 
 [!code-fsharp[Main](../../../samples/snippets/fsharp/lang-ref-1/snippet1304.fs)]
 
-You can also use a sequence expression to create a list. See "Sequence Expressions" in [Sequences](https://msdn.microsoft.com/library/6b773b6b-9c9a-4af8-bd9e-d96585c166db). For example, the following code creates a list of squares of integers from 1 to 10.
+You can also use a sequence expression to create a list. See [Sequence Expressions](sequences.md#sequence-expressions) for more. For example, the following code creates a list of squares of integers from 1 to 10.
```
Comment (mairaw) [inline docs/fsharp/language-reference/lists.md]: nit: for more what? for more information? more examples?

Comment (cartermp) [inline docs/fsharp/language-reference/lists.md]: Good catch!

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/lists?branch=pr-en-us-1105](https://review.docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/lists?branch=pr-en-us-1105)  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C3%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610032137172473-1105%5Cworkflow_report.html).

**Review by mairaw — APPROVED** [review]
LGTM!

**Pull request merged**